addons/mail/static/tests/owl/hooks/use_store_compare_keys_tests.js

```diff
@@ -0,0 +1,218 @@
+odoo.define('mail.hooks.useStoreCompareKeysTests', function (require) {
+'use strict';
+
+const { proxyComparator, proxyComparatorDeep, useStoreCompareKeys } = require('mail.hooks.useStoreCompareKeys');
+
+const { Component, QWeb, Store } = owl;
+const { xml } = owl.tags;
+
+const {
+    beforeEach: utilsBeforeEach,
+    afterEach: utilsAfterEach,
+    afterNextRender,
+    nextAnimationFrame,
+} = require('mail.messagingTestUtils');
+
+QUnit.module('mail.messaging', {}, function () {
+QUnit.module('hooks', {}, function () {
+QUnit.module('useStoreCompareKeysTests', {
+    beforeEach() {
+        utilsBeforeEach(this);
+        const qweb = new QWeb();
+        this.env = { qweb };
+        this.start = () => {
+            this.store = new Store({
+                env: this.env,
+                state: {
+                    obj: {
+                        a: 1,
+                        b: 2,
+                        c: 3,
+                    },
```

Comment:
I prefer seeing definition of `obj` in each test over sharing & hiding here.
My rule of thumb is to make important data explicit in tests.

Reply:
ok